src/qp/factory.py

```diff
@@ -20,9 +20,8 @@
 
 
 class Factory(OrderedDict):
-    """Factory that creates and mangages PDFs
+    """Factory that creates and mangages PDFs"""
```

Comment:
```suggestion
    """Factory that creates and manages PDFs"""
```

I know this wasn't really part of this PR, but I noticed along the way.